test/server/cards/02_SHD/units/RuthlessAssassin.spec.ts

```diff
@@ -0,0 +1,34 @@
+describe('Ruthless Assassin', function() {
+    integration(function(contextRef) {
+        describe('Ruthless Assassin\'s ability', function() {
+            beforeEach(function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['ruthless-assassin'],
+                        groundArena: ['greedo#slow-on-the-draw'],
+                        spaceArena: ['green-squadron-awing']
+                    },
+                    player2: {
+                        groundArena: ['wampa'],
+                    }
+                });
+            });
+
+            it('should deal 2 damage to a friendly unit.', function () {
```

Comment:
nit: friendly _ground_ unit

Reply:
that can be a space unit

Reply:
Oh yeah right I'm thinking of death trooper